frontend/coprs_frontend/coprs/views/webhooks_ns/webhooks_general.py

```diff
@@ -88,18 +88,23 @@ def add_webhook_history_record(webhook_uuid, user_agent='Not Set',
     This methods adds info of an intercepted webhook to webhook_history db
     along with the initiated build number(s).
     """
+    if user_agent is None:
+        user_agent = 'Not Set'
```

Comment:
Shouldn't we store `None` to database, instead of `Not set` string?  That would seem nicer, we may decide the default "presentation" string in HTML templates.  Also, what if clients fake the agent to "Not set" :-)

Reply:
done. we now store it in DB as `None` and present it in the template as 'Not Set'.